### ISSUE TITLE
Modifier keys accidentally can overwrite dropEffect set in useDrag

### ISSUE THREAD

**Issue by blaues0cke** [react-dnd/react-dnd#1508]
**Describe the bug**
The user is accidentally able to overwrite the `dropEffect` using the `⌘ command` when the `dropEffect` is "forced" by passing it in `options` in `useDrag`.

**Reproduction**
https://codesandbox.io/s/github/react-dnd/react-dnd/tree/gh-pages/examples_hooks_js/05-customize/drop-effects?from-embed

1. Open https://codesandbox.io/s/github/react-dnd/react-dnd/tree/gh-pages/examples_hooks_js/05-customize/drop-effects?from-embed
2. Drag the **second** box with the text `When I am over a drop zone, I have no icon` and press the modifier keys (`⌥ option` or `⌘ command`). Nothing will change as expected.
3. Drag the **first** box with the text `When I am over a drop zone, I have copy icon` and press the modifier keys (`⌥ option` or `⌘ command`). The "copy" mouse cursor will disappear while pressing `⌘ command`. This will also allowing moving where no moving should be allowed.

**Expected behavior**
I expect that in the "copy" mouse cursor will stay since the user should not be allowed to change `dropEffect` when its defined in `options`.

**Screenshots**
![Screen Recording 2019-08-09 at 12 47 30 2019-08-09 12_48_18](https://user-images.githubusercontent.com/5159398/62774035-0a329200-baa4-11e9-8d05-f3f1d79229bd.gif)

**Desktop:**

- OS: macOS Mojave 10.14.5 (18F203)
- Browser Google Chrome Version 76.0.3809.100 (Official Build) (64-bit)

**Comment by blaues0cke** [comment]
This is still an issue I would like to discuss about. :-)

**Comment by designbyadrian** [comment]
I'm not a maintainer, but I'd like to point out that it looks like you're linking the original example, not your edit?

**Comment by blaues0cke** [comment]
@designbyadrian Jep, this just happens in the original example without any changes :-)

**Comment by designbyadrian** [comment]
OIC! Well, CMD and CTRL are meta keys, so maybe it's intentional that they override the action, as CMD isn't the copy key in macOS?

The copy state doesn't change if you toggle the ALT/Option key, at least not in Safari or Chrome.

**Comment by blaues0cke** [comment]
Yes, this is another issue: The modifier key is hardcoded to "alt" but the modifier key is actually different between macOS (alt) and Windows (strg). I created this (https://github.com/react-dnd/react-dnd/issues/1517) issue for that that unfortunately has been closed automatically
